### PR TITLE
React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by Bornittah**
**Summary**
- Fetches data from the  API
- Displays a home page with different regions in the world
- Allows users to click on each region to view detailed  information
- Search bar to filter regions, countries by name

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+  width: 24em;
+  transition: all ease-in-out 0.5s;
+  margin-right: -2rem;
+}
+
+.searchButton {
+  border: none;
+  color: #646464;
+  margin-top: 0.1em;
+}
```
Comment (Whoistolu) [inline src/App.css:220]: - [ ] [REQUIRED] Kindly fix the responsiveness of the search bar. It currently looks unresponsive. Let's do this as stated by the project requirement.

### Your Design
Screenshot 2023-01-14 at 14 31 28

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+.coat-of-arm {
+  width: 28rem;
+  height: 13rem;
+}
+
+.country-list {
+  height: 100px;
+  width: 100%;
+  align-items: center;
+}
```
Comment (Whoistolu) [inline src/App.css:147]: - [ ] [REQUIRED] Well done with the styles. However, you need to follow the design pattern provided. If you take a look at the color pattern, you will see that there are two shades of a particular colour applied but you only added just one shade. Please implement this on the Homepage and the Details page.
I can recommend using [CSS Linear gradient](https://www.w3schools.com/css/css3_gradients.asp) to achieve this or the [Column-Rule-Selector](https://developer.mozilla.org/en-US/docs/Web/CSS/column-rule-color)

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot 2023-01-14 at 14 31 28 2 | Screenshot 2023-01-14 at 14 48 10

### Your Design (Homepage)
Screenshot 2023-01-14 at 14 32 37

File: README.md [hunk 3]
```diff
@@ -1 +1,187 @@
...
+- Allows users to click on each region to view detailed  information
+- Search bar to filter regions, countries by name
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://countries-api-8kc4.onrender.com/)
```
Comment (Whoistolu) [inline README.md]: - [ ] [REQUIRED] Great job deploying this project 👍🏽  However, the deployment link is broken. Kindly fix it and ensure it is properly deployed before asking for another review.

Screenshot 2023-01-14 at 14 33 59

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Bornittah 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Bornittah,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- No linter errors ✔️
- Professional readme file ✔️
- The deployed application works fine ✔️


209362895-9762f2f8-13cf-4ce0-90b6-a672c4f04196
# Status: Approved ✔️✔️✔️
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**